baremetal/metal3cluster_manager_test.go

```diff
@@ -316,9 +318,29 @@ var _ = Describe("Metal3Cluster manager", func() {
 					},
 				},
 			},
+			OwnerRef:            bmcOwnerRef,
 			ExpectError:         false,
 			ExpectedDescendants: 1,
 		}),
+		Entry("One Cluster Descendant with error OwmRef", descendantsTestCase{
```

Comment:
we could improve this message a bit, i.e 
```suggestion
		Entry("One Cluster Descendant with wrong OwnerRef", descendantsTestCase{
```

and in the previous entry to make the difference a bit clear:
```suggestion
		Entry("One Cluster Descendant with proper OwnerRef", descendantsTestCase{
```

Reply:
Updated, PTAL.